content/en/developers/ide_plugins/visual_studio/_index.md

```diff
@@ -37,6 +37,16 @@ Analyze and improve the performance of your applications with real-time profilin
 
 {{< img src="/developers/ide_plugins/visual_studio/top-list.png" alt="The Code Insights view">}}
 
+### Static Analysis
+
+The Datadog extension runs [Static Analysis][14] rules on the source files you have open in your Solution. The goal is to detect and fix problems such as maintainability issues, bugs, or security vulnerabilities in your code before you commit your changes.
+
+Static Analysis supports scanning across multiple programming languages. For a complete list, see [Static Analysis Rules][15]. For file types belonging to supported languages, issues are shown in the source code editor using Visual Studio's code inspection system, and suggested fixes can be applied directly:
+
+{{< img src="/developers/ide_plugins/visual_studio/TBD.png" alt="A static analysis rule violation and recommended fix." style="width:100%;" >}}
```

Comment:
To be replaced: image in dark mode @markazerdd

Reply:
@AlonAm to add the screenshot in here

Reply:
Added a new screenshot, let me know what you think, thanks